docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+      - [Token Symbol](#token-symbol)
+      - [Gas Configuration](#gas-configuration)
+      - [Airdropping Native Tokens](#airdropping-native-tokens)
+      - [Adding a Custom Precompile To Modify the EVM](#adding-a-custom-precompile-to-modify-the-evm)
+        - [Native Minting](#native-minting)
+        - [Configure Contract Deployment Whitelist](#configure-contract-deployment-whitelist)
+        - [Configure Transaction Allow List](#configure-transaction-allow-list)
+  - [Deploying the Subnet](#deploying-the-Subnet)
+- [Understanding the Genesis File](#understanding-the-genesis-file)
+    - [Config](#config)
+      - [`chainId`](#chainid)
+      - [Hardforks](#hardforks)
+    - [Fee Config](#fee-config)
+      - [`gasLimit`](#gaslimit)
+      - [`targetBlockRate`](#targetblockrate)
+      - [`minBaseFee`](#minbasefee)
+      - [`targetGas`](#targetgas)
+      - [`baseFeeChangeDenominator`](#basefeechangedenominator)
+      - [`minBlockGasCost`](#minblockgascost)
+      - [`maxBlockGasCost`](#maxblockgascost)
+      - [`blockGasCostStep`](#blockgascoststep)
+    - [Validator Fee Recipient](#validator-fee-recipient)
+    - [Precompiles](#precompiles)
+      - [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig)
+      - [`contractNativeMinterConfig`](#contractnativeminterconfig)
+      - [`txAllowListConfig`](#txallowlistconfig)
+    - [Genesis Block](#genesis-block)
+      - [`nonce`](#nonce)
+      - [`timestamp`](#timestamp)
+      - [`extraData`](#extradata)
+      - [`gasLimit`](#gaslimit-1)
+      - [`difficulty`](#difficulty)
+      - [`mixHash`](#mixhash)
+      - [`coinbase`](#coinbase)
+      - [`parentHash`](#parenthash)
+      - [`gasUsed`](#gasused)
+      - [`number`](#number)
+      - [`airdropHash` `baseFeePerGas` `airdropAmount`](#airdrophash-basefeepergas-airdropamount)
+    - [Native Token Allocation](#native-token-allocation)
+- [Creating a Custom Genesis File](#creating-a-custom-genesis-file)
+  - [Setting the Chain Id](#setting-the-chain-id)
+  - [Configuring Fees and Gas](#configuring-fees-and-gas)
+  - [Allocating Native Tokens](#allocating-native-tokens)
+  - [Minting Native Tokens](#minting-native-tokens)
+  - [Restricting Contract Deployers](#restricting-contract-deployers)
+- [Creating and Deploying the Subnet Using the Custom Genesis File](#creating-and-deploying-the-Subnet-using-the-custom-genesis-file)
+- [Deploying the Game Contract](#deploying-the-game-contract)
+  - [Metamask Configuration](#metamask-configuration)
+  - [Metamask Gas Configuration](#metamask-gas-configuration)
+  - [Deploying the Contract](#deploying-the-contract)
+    - [NativeMinterInterface.sol](#nativeminterinterfacesol)
+    - [Game.sol](#gamesol)
+    - [Compiling and deploying](#compiling-and-deploying)
+  - [Setting Game Contract as a Minter](#setting-game-contract-as-minter)
+  - [Play the Game](#play-the-game)
+- [Conclusion](#conclusion)
+  - [Resources](#resources)
+
+## Prerequisites
+
+### Metamask
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)